controlplane/kubeadm/internal/workload_cluster_conditions_test.go

```diff
@@ -694,7 +708,14 @@ func TestUpdateEtcdConditions(t *testing.T) {
 				KCP:      tt.kcp,
 				Machines: collections.FromMachines(tt.machines...),
 			}
-			w.UpdateEtcdConditions(ctx, controlPane)
```

Comment:
WDYT about preserving the test coverage for UpdateEtcdConditions and adding an additional t.Run with this?

Just easy to lose test coverage if the test code diverges over time from UpdateEtcdConditions

Reply:
Added (created a new one, cleaned up the old ones)